xbrowse_server/base/management/commands/add_project.py

```diff
@@ -1,14 +1,43 @@
 from django.core.management.base import BaseCommand
 from xbrowse_server.base.models import Project
 import sys
+from xbrowse_server.phenotips.utilities import add_new_user_to_phenotips
+from xbrowse_server.phenotips.utilities import get_uname_pwd_for_project
+from django.conf import settings
 
 class Command(BaseCommand):
 
     def handle(self, *args, **options):
+        if len(args)<2 or not args[0] or not args[1]:
+          print '\n'
+          print 'please provide a project ID and a project name as the first two arguments. If project name as spaces, please remember to enclose in double quotes'
+          print 'example: python manage.py 1kg "1000 Genomes Project"\n'
+          sys.exit()
         project_id = args[0]
+        project_name = args[1]
```

Comment:
It's given as part of the "first name" of the username(s) created for that project in Phenotips. You should be able to see it in the top right hand corner